Packs/FeedCrowdstrikeFalconIntel/Integrations/FeedCrowdstrikeFalconIntel/FeedCrowdstrikeFalconIntel.py

```diff
@@ -0,0 +1,273 @@
+            params = self.get_last_modified_time()
+
+        suffix_url_to_filter_by = self.build_url_suffix(params, actors_filter)
+        response = self.check_quota_status(suffix_url_to_filter_by)
+        parsed_indicators = self.create_indicators_from_response(response, feed_tags)  # list of dict of indicators
+
+        # for get_indicator_command only
+        if limit:
+            parsed_indicators = parsed_indicators[int(offset): int(offset) + int(limit)]
+        return parsed_indicators
+
+    def set_last_modified_time(self):
+        current_time = datetime.now()
+        current_timestamp = datetime.timestamp(current_time)
+        timestamp = str(int(current_timestamp))
+        integration_context_to_set = {'last_modified_time': timestamp}
+        demisto.setIntegrationContext(integration_context_to_set)
+
+    def get_last_modified_time(self):
+        demisto.setIntegrationContext(None)
+        integration_context = demisto.getIntegrationContext()
+        if not integration_context:
+            params = ''
+            self.set_last_modified_time()
+        else:
+            last_modified_time = demisto.getIntegrationContext()
+            relevant_time = int(last_modified_time['last_modified_time'])
+            params = f"last_modified_date%3A%3E{relevant_time}"
+            self.set_last_modified_time()
+        return params
+
+
+def module_test_command(client: Client, args: dict, feed_tags: list):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client(Client): CrowdStrike Feed client
+        args(Dict): The instance parameters
+        feed_tags: The indicator tags
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    try:
+        client.build_iterator(argToList(demisto.params().get('feedTags')), 1, 0)
+    except Exception:
+        raise Exception("Could not fetch CrowdStrike Feed\n"
+                        "\nCheck your API key and your connection to CrowdStrike.")
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: dict, feed_tags: list) -> Tuple[str, dict, list]:
+    """Initiate a single fetch-indicators
+
+    Args:
+        client(Client): The CrowdStrike Client.
+        args(dict): Command arguments.
+        feed_tags: The indicator tags
+    Returns:
+        str, dict, list. the markdown table, context JSON and list of indicators
+    """
+    offset = int(args.get('offset', 0))
+    limit = int(args.get('limit', 150))
+    target_countries = args.get('target_countries') if args.get('target_countries') \
+        else demisto.params().get('target_countries')
+    target_industries = args.get('target_industries') if args.get('target_industries') \
+        else demisto.params().get('target_industries')
+    custom_filter = args.get('custom_filter') if args.get('custom_filter') \
+        else demisto.params().get('custom_filter')
```

Comment:
Because this command is for testing, we want to check what will happen in fetch indicators, so if the user did not enter arguments we will want to check if there are parameters he did enter into the fetch indicators and if so we put them in this command.